dep_index.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
remove the `.Rproj` file from the commit since its not part of the code required to make the dataset